src/python/turicreate/test/test_recsys_api.py

```diff
@@ -8,11 +8,13 @@
 from __future__ import absolute_import as _
 import unittest
 import sys
+import pytest
 import turicreate as tc
 
 DELTA = 0.000001
 
 
+@pytest.mark.minimal
```

Comment:
This shouldn't be here. I believe I told you that already in the previous pull request.

Reply:
Well, I need to remove it manually.